tools/docker/builder/openwrt/Dockerfile

```diff
@@ -0,0 +1,100 @@
+ARG HOME=/home/openwrt
+
+####
+## OpenWrt pre-requisites and "openwrt" user
+####
+FROM ubuntu:18.04 as openwrt-prerequisites
+
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+    build-essential libncurses5 libncurses5-dev python python3 \
+    unzip git ca-certificates gawk wget file bash \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN useradd -ms /bin/bash openwrt
+
+####
+## Pre-built OpenWrt (and extracted sdk)
+####
+FROM openwrt-prerequisites as openwrt-prebuilt
+
+ARG HOME
+COPY --chown=openwrt:openwrt configs/sdk_config $HOME/sdk_config
+USER openwrt
+
+ARG OPENWRT_REPOSITORY=https://git.prpl.dev/prplwrt/prplwrt.git
+ARG OPENWRT_VERSION=9d2efd
+ARG TARGET=mvebu
+
+WORKDIR $HOME
+
+RUN git clone "$OPENWRT_REPOSITORY" openwrt \
+    && cd openwrt \
+    && git checkout "$OPENWRT_VERSION"
+
+WORKDIR $HOME/openwrt
+
+RUN cp "${HOME}/sdk_config" .config \
```

Comment:
Since sdk_config is just a oneliner, and we definitely always want that one line in the config, I'd do it with echo here instead of requiring a separate file. If it becomes more than that one line, we can still make it a file.

Reply:
Done.